rust_src/src/lisp.rs

```diff
@@ -351,6 +351,19 @@ impl From<LispObject> for bool {
     }
 }
 
+impl From<LispObject> for char {
```

Comment:
I just remembered: we already did this but not for the `char` type directly. We occasionally have to use values outside the normal unicode range, so we have a `Codepoint` type in multibyte.rs to use instead.